### ISSUE TITLE
I got a plot that don't look like an  inverted cone. Is it Ok?

### ISSUE THREAD

**Issue by alqurri77** [marcellodebernardi/loss-landscapes#7]
I got the below plot. Is it correct because I was expecting a shape like an inverted cone?

<img width="178" alt="sam_loss_surface" src="https://github.com/marcellodebernardi/loss-landscapes/assets/75499343/58dc26d9-e9e9-4d83-9102-c0744fd546cd">

**Comment by ashishjv1** [comment]
Which model and dataset did you train?

**Comment by alqurri77** [comment]
This is for a custom model that I made which uses Transformers for segmentation. The dataset is  [synapse](https://www.synapse.org/#!Synapse:syn3193805/wiki/217789)

**Comment by ashishjv1** [comment]
well atleast from what I saw on the original paper the loss surface should look like an inverted cone. I trained Resnet18 on CIFAR10 and my results are not even close.

**Comment by alqurri77** [comment]
You mean your result does not look like the one in the paper?

**Comment by ashishjv1** [comment]
Yes

**Comment by alqurri77** [comment]
Do you have any clue why that happened.

**Comment by ashishjv1** [comment]
Try this repo - https://github.com/tomgoldstein/loss-landscape

**Comment by alqurri77** [comment]
Thank you...Could I use the above repo for my custom model, or only for pre-specified models?

**Comment by ashishjv1** [comment]
You can use your custom model; however, you will need to import your custom model inside plot_surface.py or add your custom model to model_loader.py. You will also need a pre-trained checkpoint (Usually *.pth or .t7) file which needs to be specified using --model_file argument.

**Comment by alqurri77** [comment]
OK. Thank you. I will try that.
